frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ManageServingRuntimeModal.tsx

```diff
@@ -64,16 +66,17 @@ const ManageServingRuntimeModal: React.FC<ManageServingRuntimeModalProps> = ({
 
   const tokenErrors = createData.tokens.filter((token) => token.error !== '').length > 0;
 
+  const resourceIsSet = (fieldName, min = 1) => {
+    const requests = createData.modelSize.resources.requests?.[fieldName];
+    const limits = createData.modelSize.resources.limits?.[fieldName];
+    return (
+      requests && limits && min <= parseInt(requests) && parseInt(requests) <= parseInt(limits)
```

Comment:
The old check wasn't allowing the resource == limit.  Setting them both the same seems a pretty common use case, but let me know if this seems like a problem